### ISSUE TITLE
Add some features from notion

### ISSUE THREAD

**Issue by wilhelmy** [paperwm/PaperWM#378]
Coming from notion, I decided to give paperwm a spin because it seems like a good successor project.
There are several keyboard based features I miss in paperwm that make notion a very efficient tool for me:

* mark-window-for-action/attach workflow (especially useful in multi-monitor scenarios). This could be especially useful with generalized "vim-style" keybindings where you perform actions on... such as mark and then close or mark and put into the same "tiling area"
* goto window by title/attach window by title
* integration with a query dialog such as rofi
* moving focus between screens and taking the mouse cursor along

Are there any plans to support these, or opinions on these features?
Thanks for the interesting approach to wayland compositor development by just extending gnome rather than doing it from scratch. It makes so much sense!

**Comment by hedning** [comment]
Yep, we've thought about adding mark + actions, `Super+T` is somewhat inspired by the concept, but could use some love.

For go to window, I'd reccomend the switcher exension: https://github.com/daniellandau/switcher/, though it doesn't do attach, which would be nice.

IIRC we've looked a bit at getting something to work with rofi. We have a gnome-shell-mode for emacs which uses dbus to push «live» code, so something like that could work fairly well.

The default `Super+Shift+Arrow` bindings moves focus between screens and warps the cursor (warping should now work in Wayland too IIRC), though you might want to disable the corresponding native gnome bindings as they're configured per window bindings which we're forced inherit if active, leaving the shortcut disabled on empty workspaces.
